contracts/APPCoin.sol

```diff
@@ -59,7 +60,21 @@ contract APPCoin is Initializable, ERC777Upgradeable, PausableUpgradeable, Ownab
             delete frozenMap[acc];
         }
     }
+    function charge(address account, uint256 amount, bytes memory data) public onlyOwner {
```

Comment:
`onlyAppOwner`?

Reply:
It depends on the implementation of billing service. May change later.